src/bin/cargo-info/command/info.rs

```diff
@@ -20,6 +22,14 @@ fn info_subcommand() -> Command {
         )
         .arg_index("Registry index URL to search packages in")
         .arg_registry("Registry to search packages in")
+        .arg(
+            Arg::new("path")
+                .long("path")
+                .action(ArgAction::Set)
+                .value_name("PATH")
+                .help("Filesystem path to local crate to inspect")
+                .group("selected"),
```

Comment:
It seems this does not work.

Reply:
What is it you are trying to do?

Reply:
Sorry for the confusion. I mixed up the help_heading and group API. It works well. And it seems we don't need the group here.